### PR TITLE
Added value accessors to support Angular2 ngModel.

### PR DESCRIPTION
**Pull request by nsndeck**

### CODE REVIEW

**Comment by vakrilov** [comment]
:+1:

File: src/nativescript-angular/value-accessors/time-value-accessor.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import {Directive, ElementRef, Renderer, Self, forwardRef, Provider} from 'angular2/core';
+import {NG_VALUE_ACCESSOR, ControlValueAccessor} from 'angular2/src/common/forms/directives/control_value_accessor';
+import {isBlank, CONST_EXPR} from 'angular2/src/facade/lang';
+
+const TIME_VALUE_ACCESSOR = CONST_EXPR(new Provider(
```
Comment (hdeshev) [inline src/nativescript-angular/value-accessors/time-value-accessor.ts]: CONST_EXPR not actually needed. AFAIK Angular needs it for Dart compatibility.

**Comment by hdeshev** [comment]
:+1: after getting rid of `CONST_EXPR` occurrences. :neckbeard:

**Comment by hdeshev** [comment]
Cheated BaseValueAccessor with a generic `TView` param, so we can test `writeValue` without a UI tree.

**Pull request merged**